packages/gatsby-theme-api-docs/src/components/info-value.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import {
+  Markdown,
+  useISO310NumberFormatter,
+  designSystem,
+} from '@commercetools-docs/ui-kit';
+
+const InfoValue = (props) => {
+  const value = props.children;
+  const valueType = props.type || typeof value;
+  const formatNumber = useISO310NumberFormatter();
+
+  const customCodeStyle = css`
+    font-size: ${designSystem.typography.fontSizes.small};
+  `;
+
+  switch (valueType) {
+    case 'boolean':
+      // We want to show the actual boolean value for the query parameter defaults.
+      return props.isQueryParameter ? (
+        <>
```

Comment:
I think also this react fragment is not needed @timonrey

Reply:
Right 👍🏼

Reply:
@gabriele-ct One thing I saw now in the snapshots...we have the uniqueItems info that does not have any value...we would render it with a colon like this so I think it's better that the keep it as it was...that the component handles the colon.

<img width="370" alt="Screenshot 2024-07-04 at 10 01 48" src="https://github.com/commercetools/commercetools-docs-kit/assets/31767369/e21da2ae-fed6-4cde-b82d-3562d8527503">